### PR TITLE
[IMP] mail: adds shadow to RPCs

### PR DESCRIPTION
**Pull request by ThanhDodeurOdoo**
This commit adds shadow to RPCs that are not supposed to be blocking.

task-2352703

### CODE REVIEW

File: addons/mail/static/src/models/thread/thread.js [hunk 1]
```diff
@@ -383,7 +383,7 @@ function factory(dependencies) {
                         isMobile: device.isMobile,
                     }),
                 },
-            });
+            }, { shadow: true });
```
Comment (seb-odoo) [inline addons/mail/static/src/models/thread/thread.js:386]: I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.

File: addons/mail/static/src/models/thread/thread.js [hunk 2]
```diff
@@ -446,7 +446,7 @@ function factory(dependencies) {
...
```
Comment (seb-odoo) [inline addons/mail/static/src/models/thread/thread.js:449]: I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.

File: addons/mail/static/src/models/thread/thread.js [hunk 3]
```diff
@@ -552,7 +552,7 @@ function factory(dependencies) {
                     partner_ids: [this.env.messaging.currentPartner.id],
                     context: {}, // FIXME empty context to be overridden in session.js with 'allowed_company_ids' task-2243187
                 },
-            }));
+            }, { shadow: true }));
```
Comment (seb-odoo) [inline addons/mail/static/src/models/thread/thread.js:555]: I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.

File: addons/mail/static/src/models/activity/activity.js [hunk 4]
```diff
@@ -167,7 +167,7 @@ function factory(dependencies) {
                     attachment_ids: attachmentIds,
                     feedback,
                 },
-            }));
+            }, { shadow: true }));
```
Comment (seb-odoo) [inline addons/mail/static/src/models/activity/activity.js:170]: I would revert this one, this is a direct user action for which we don't handle "loading" ourselves.

**Review by alexkuhn — APPROVED** [review]
LGTM, squash then merge

**Comment by seb-odoo** [comment]
use tag `[FIX]`

**Comment by ThanhDodeurOdoo** [comment]
closed in favor of https://github.com/odoo/odoo/pull/59991

**Pull request closed without merging**